Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM node:17
```

Comment:
It just so happens that node:17 ships with python 3.9.2 which is nice - if we ever feel the need to upgrade our python version then we might need to do some pyenv stuff.